package.json

```diff
@@ -33,9 +33,9 @@
     "postcss-values-parser": "^6.0.2"
   },
   "devDependencies": {
-    "chai": "^4.3.4",
+    "chai": "^4.3.7",
     "mocha": "^9.1.3",
```

Comment:
Have you created a ticket for updating `mocha 10.2.0`? If not, please do.

Reply:
OK, WRP-11943